### DISCUSSION TITLE
add github topics

### DISCUSSION THREAD

**Discussion by paressuex11** [community/community#62100] [Mobile]
idont see any topics of repo while using apps

**search with topics**
**topics in star lists**

**Comment by mxie** [comment]
Hi @paressuex11 - thanks for your feedback! This is feature that's in our backlog but it's not currently on our roadmap. We'd gladly revisit this request if there's a higher demand.

In the meantime, can you tell me more about how you like to use the Topics feature? Otherwise, I recommend checking out the Explore tab, which may provide what you need.

**Reply by Alejandrocontrera** [reply]
4152314056478923

**Reply by Alejandrocontrera** [reply]
Está  boleada y la quiero desbloquear es de banco BBVA
